src/core/Header/Header.stories.tsx

```diff
@@ -22,11 +22,16 @@ const baseArgs: HeaderProps = {
   searchButton: <Icon name="icon-gui-magnifying-glass-outline" size="1.5rem" />,
   sessionState: {
     signedIn: false,
+    isDropDownAccountLinks: false,
     account: {
+      companyName: "Ably",
```

Comment:
I have to check - have you verified that this field exists? I could be mistaken, but after a quick search for this field elsewhere I can't see any signs of it.

Reply:
thanks,  will replicate what the object being passed in the `MeganavSessionState` of the current Meganav